CRM/Core/xml/Menu/Admin.xml

```diff
@@ -580,7 +580,21 @@
   <item>
      <path>civicrm/admin/options/grant_type</path>
      <title>Grant Types</title>
-     <desc>List of types which can be assigned to Grants. (Enable CiviGrant from Administer > Systme Settings > Enable Components if you want to track grants.)</desc>
+     <desc>List of types which can be assigned to Grants. (Enable CiviGrant from Administer > System Settings > Enable Components if you want to track grants.)</desc>
+     <page_callback>CRM_Admin_Page_Options</page_callback>
+     <adminGroup>Option Lists</adminGroup>
+     <weight>385</weight>
+  </item>
+    <item>
+     <path>civicrm/admin/options/grant_status/edit</path>
```

Comment:
Should this not civi in the civigrant extension?

Reply:
thanks, this has been moved to the extension now.